auxcov/src/lib.rs

```diff
@@ -100,3 +102,5 @@ fn stop_code_coverage(coverage_file: Value) {
 
 	result
 }
+
+pub fn anti_dce_stub() {}
```

Comment:
I spent all night trying to figure out why it wasn't working and it turns out the entire crate was getting DCE'd.

This function needs to be called by the consumer to prevent that.

I have 0 clue why this is, it must be a bug in `inventory`.